src/en/rinkocomics/src/eu/kanade/tachiyomi/extension/en/rinkocomics/RinkoComics.kt

```diff
@@ -0,0 +1,400 @@
+import java.util.Locale
+
+class RinkoComics : HttpSource(), ConfigurableSource {
+
+    override val name = "Rinko Comics"
+
+    override val baseUrl = "https://rinkocomics.com"
+
+    override val lang = "en"
+
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient
+
+    private val preferences by getPreferencesLazy()
+
+    private var genresList: List<Genre> = emptyList()
+
+    override fun headersBuilder() = super.headersBuilder()
+        .add("Referer", "$baseUrl/")
+
+    override fun popularMangaRequest(page: Int): Request =
+        GET(comicsUrl(page).build(), headers)
+
+    override fun popularMangaParse(response: Response): MangasPage =
+        parseComicsPage(response)
+
+    override fun latestUpdatesRequest(page: Int): Request =
+        GET(
+            comicsUrl(page)
+                .addQueryParameter(SORT_PARAM, SortFilter.OPTIONS.first().second)
+                .build(),
+            headers,
+        )
+
+    override fun latestUpdatesParse(response: Response): MangasPage =
+        parseComicsPage(response)
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = comicsUrl(page)
+            .addQueryParameter("post_type", "comic")
+
+        if (query.isNotBlank()) {
+            url.addQueryParameter("s", query)
+        }
+
+        filters.forEach { filter ->
+            when (filter) {
+                is GenreFilter -> {
+                    filter.state.filter { it.state }
+                        .forEach { url.addQueryParameter("genres[]", it.slug) }
+                }
+                is SortFilter -> {
+                    filter.toQuery()?.let { url.addQueryParameter(SORT_PARAM, it) }
+                }
+                else -> {}
+            }
+        }
+
+        return GET(url.build(), headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage =
+        parseComicsPage(response)
+
+    override fun mangaDetailsParse(response: Response): SManga =
+        parseMangaDetails(response.asJsoup())
+
+    private fun parseMangaDetails(document: Document): SManga = SManga.create().apply {
+        title = requireField(
+            document.selectFirst(".comic-info-upper h1")?.text()
+                ?: document.selectFirst("h1")?.text(),
+            "title",
+        )
+
+        thumbnail_url = document.selectFirst("meta[property=og:image]")?.attr("content")
+
+        val authors = document.select(".comic-graph > span")
+            .map { it.text().trim() }
+            .filter { it.isNotBlank() && it != "•" }
+            .distinct()
+
+        author = authors.firstOrNull()
+        artist = authors.getOrNull(1)
+
+        status = parseStatus(
+            document.selectFirst(".comic-status span:last-child")?.text(),
+        )
+
+        genre = document.select(".comic-genres .genres .genre")
+            .joinToString(", ") { it.text().trim() }
```

Comment:
`", "` is the default and can be omitted.
```suggestion
            .joinToString() { it.text() }
```